langchain/prompts/loading.py

```diff
@@ -52,23 +52,29 @@ def _load_template(var_name: str, config: dict) -> dict:
     return config
 
 
-def _load_examples(config: dict) -> dict:
-    """Load examples if necessary."""
-    if isinstance(config["examples"], list):
-        pass
-    elif isinstance(config["examples"], str):
-        with open(config["examples"]) as f:
-            if config["examples"].endswith(".json"):
+def _load_examples(var_name: str, config: dict) -> dict:
```

Comment:
I was wondering if you see any advantages to parametrize the method with `var_name`? Can we expect that we call it with a different value than `var_name="examples"`?

Reply:
I agree with what you're saying. I didn't like doing it this why but was matching how the template was loaded.

https://github.com/hwchase17/langchain/blob/8031b6572fafc8ab8aa80735a8cceed0d3408a5f/langchain/prompts/loading.py#L33

I think those methods can probably be combined into a single "loader" method but I felt that was too far for the scope of this PR

Reply:
That's what I thought :-) But I think the `_load_template` method is a different case, because we are indeed reusing it with different parameters vaues, i.e. `suffix`, `prefix` and `template`.

Reply:
Thanks for that. I updated it so it is purely loading examples without any parametrization.